adapters/repos/db/vector/hnsw/index.go

```diff
@@ -293,9 +304,13 @@ func New(cfg Config, uc ent.UserConfig,
 		store:                  store,
 		allocChecker:           cfg.AllocChecker,
 		visitedListPoolMaxSize: cfg.VisitedListPoolMaxSize,
+
+		docIDVectors: make(map[uint64][]uint64),
 	}
 	index.acornSearch.Store(uc.FilterStrategy == ent.FilterStrategyAcorn)
 
+	index.multivector.Store(uc.Multivector.Enabled)
+
 	if uc.BQ.Enabled {
 		var err error
 		index.compressor, err = compressionhelpers.NewBQCompressor(
```

Comment:
Are we planning to support compression with multi-vector?

Reply:
The compression is supported only for the BQ